Source/SpaceparkAPI/Objects/Person.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace SpaceparkAPI.Objects
+{
+    public class Person
```

Comment:
```suggestion
    public record Person
```
C# 0.9 introduces record
https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/records
This keyword is prefered when making model-type classes - value equality is autohandled for instance.

Reply:
Interesting, I did not know that.

Reply:
But Im not sure if Person is a model object. We are only using Parkings and Spaceport as models.